docs/intro/tutorial04.md

```diff
@@ -89,6 +89,8 @@ Brigade uses simple JavaScript files to run tasks. When it comes to task running
 Given this, the role of the `brigade.js` file is to declare event handlers. And it's easy. Open `brigade.js` and write this JavaScript code into it:
 
 ```javascript
+const { events, Job } = require("brigadier");
```

Comment:
I think this can be simplified to

```
const { events } = require("brigadier");
```

Because the job object isn't actually added or used until the following step.

Reply:
Good call; updated.